.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "external-secrets"]
+	path = external-secrets
+	url = https://github.com/openshift/external-secrets.git
+	branch = main
```

Comment:
submodule branch must be same as parent branch. Re-initialize submodules with `release-0.14` branch.

Reply:
can be resolved.